virtual-table-UI/src/app/first-level/first-level.component.html

```diff
@@ -1,9 +1,9 @@
 <div class="container">
-    <div *ngFor="let item of realdata | keyvalue; let i = index">
-      <div class="level1Unit" (click)="level1Click(item)">
-        <div [id]="'viewport'+i" class="level1Image"></div>
-        {{item.value.proteinName}}
-      </div>
+  <div *ngFor="let item of realdata | keyvalue; let i = index">
+    <div class="level1Unit" (click)="level1Click(item)" >
+      <img src='{{"/www/virtualflow-covid.hms.harvard.edu/docroot/covid19_compoundtable/virtual-table-UI/dist/virtual-table-UI/Structures/"+ item.proteinName +"/Receptor.png"}}' class="level1Image">
```

Comment:
The absolute URL should look something like:

https://virtualflow-covid.hms.harvard.edu/Structures/mpro_s1_ds1/Receptor.png

The current value is an absolute path in the file system of their server.

Reply:
For attributes in Angular we usually use `[src]`. So the final result should be:

```html
<img [src]="'https://virtualflow-covid.hms.harvard.edu/Structures/' + item.proteinName + '/Receptor.png'">
```